BrowserKit/Sources/WebEngine/EngineTelemetryProxy.swift

```diff
@@ -24,6 +24,12 @@ public enum EngineTelemetryEvent {
 
     // A page load was completed.
     case pageLoadFinished
+
+    /// Sends an event for ads found on the page. Includes the provider name and ad URLs.
+    case trackAdsFoundOnPage(String, [String])
+
+    /// Sends an event for ads clicked on the page. Includes the provider name.
+    case trackAdsClickedOnPage(String)
```

Comment:
I don't remember, can we have named params inside an enum? Would make it easier to know what those are

Reply:
Yes, and I agree 💯 I'll update these to include labels